recipes/install-elasticsearch.rb

```diff
@@ -0,0 +1,105 @@
+# Cookbook Name:: mh-opsworks-recipes
+# Recipe:: install-elasticsearch
+
+::Chef::Recipe.send(:include, MhOpsworksRecipes::RecipeHelpers)
+::Chef::Recipe.send(:include, MhOpsworksRecipes::DeployHelpers)
+
+elk_info = get_elk_info
+
+es_major_version = elk_info[:es_major_version]
+es_version = elk_info[:es_version]
+es_cluster_name = elk_info[:es_cluster_name]
+data_path = elk_info[:es_data_path]
+es_heap_size = xmx_ram_for_this_node(0.5)
+
+apt_repository 'elasticsearch' do
+  uri "http://packages.elasticsearch.org/elasticsearch/#{es_major_version}/debian"
+  components ['stable', 'main']
+  keyserver 'ha.pool.sks-keyservers.net'
+  key '46095ACC8548582C1A2699A9D27D666CD88E42B4'
+end
+
+include_recipe "mh-opsworks-recipes::update-package-repo"
+install_package("elasticsearch=#{es_version}")
```

Comment:
The second approach on that page seems to imply you can get compression without logrotate but I'm not entirely sure.